readme.warpos

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Probably this file should live in Packaging/WarpOS or Packaging/Amiga

Reply:
Probably but that's why I did a fork: To not mess with the original before a proper final port is done, meanwhile people can try it and give some input. My changes don't use 8bit screen mode as this doesn't work in my environment (wrong colors and unable to do fullscreen). I even did a unique Makefile for the whole project instead of cmake.........Just take what could be relevant meanwhile I would try to do the moving target that is a MOS binary with cmake.

Reply:
Colors are fixed by this commit https://github.com/diasurgical/devilutionX/commit/319bdeddd242bb79c5145fd81ee68d584d4b220e

Reply:
You're correct but there's a problem: In window mode, besides not so good colors when you have a 16 bit desktop, movie starts with his own resolution not (scaled for) 640x480 as menu has when appears. But when you select fullscreen in diablo.ini, game crashes possibly due to the inability in my system to choose that small movie screen resolution. In non 8 bit mode, movies are scaled to the proper game resolution in window and fullscreen.

Reply:
@Cowcat5150 when executing SDL_SetVideoMode, if the requested screenmode doesn't exist, it should default to the lowest possible/available, it shouldn't crash. I would issue this as a WarpOS SDL bug.

Reply:
But the movie playing in 8bit mode should be scaled to 640x480? Crash or not is another kind of animal when warpos is running in MOS. I don't know at this moment.

Reply:
No, the resolution changes to 320x200 because it takes less resources from a low end system than upscaling to 640x480. 8bit mode in devilitionX is “low end system mode”. And 8bit screen also takes less memory bandwidth. There’s nothing in the game as is that is more than 256 colors either.

Reply:
Morphos by default has the lower screen limit set to 640x480 for all bit depths. I will test what happens when you have Ambient set to only 8bit without having game hardcoded to 8bit or if there's a way to make a Radeon promote a 320x200 mode.

Reply:
If 640x480 is lowest, that resolution is picked. However there’s no scaling so the video will be centered and small

Reply:
Should be. It's a matter to test that when a MOS port will exist.